### ISSUE TITLE
Wher do you keep you models?

### ISSUE THREAD

**Issue by Andem6** [BahamutoD/BDArmory#49]
I wanted to do a quick retexture of you'r parts, but I can't find the models. help?

**Comment by Shuudoushi** [comment]
They're found in the parts folder, they're simply 'model.mu'.

**Comment by Andem6** [comment]
Yes, well, I can't open .mu files. That's why I'm looking on Github for the non-.mu files.

**Comment by Shuudoushi** [comment]
Models aren't part of source code 90% of the time... So the likely hood of finding non-.mu versions is almost none. There is a blender plug-in on the forums for opening .mu and .craft files in blender though.
